climpred/bias_reduction.py

```diff
@@ -0,0 +1,92 @@
+import logging
+
+import xarray as xr
+
+from climpred.metrics import Metric
+
+
+def _mean_bias_reduction_quick(hind, bias, dim):
+    """Quick reduction of mean bias over all initializations.
+
+    Args:
+        hind (xr.object): hindcast.
+        bias (xr.object): bias.
+        dim (str): Time dimension name in bias.
+
+    Returns:
+        xr.object: bias reduced hind
+
+    """
+    bias_reduced_hind = hind - bias.mean(dim)
+    return bias_reduced_hind
+
+
+def _mean_bias_reduction_properly(hind, bias, dim):
```

Comment:
I would call this `_proper_mean_bias_reduction` or `_cross_validated_mean_bias_reduction`.

Reply:
Cross